kibbeh/src/ui/UserBadgeLg.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import { SolidDogenitro } from "../icons";
+import Logo from "../icons/LogoIcon";
+
+export interface UserBadgeLgProps extends React.HTMLProps<HTMLDivElement> {
+  icon: "logo" | "dogeNitro" | "dogeStaff" | "dogeContributor";
+}
+
+// @todo the tag doesn't really glow like in figma right now
```

Comment:
wdym by glow? i dont see any glow here https://www.figma.com/file/CS01VVLR7ArQl0afYFkNj3/Web-App?node-id=8277%3A990

Reply:
Not this line again....

Reply:
Fixed